docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0154E":"(%s) is not a supported architecture for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0155E":"(%s) is not a supported endpoint for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_url_) is not a supported endpoint for _componentName_. Skipping _pluginid_... Supported: _urls_
 
   **Action:**
 
-  TODO
+  Refer to component API doc.
```

Comment:
i dont think users will know what you mean by this.
users need to confirm they're on the right version of zowe and the right version of the plugin, and then refer to the plugin documentation or contact the plugin developer for more info.